hydradx/notebooks/Omnipool/vault/ImpermanentLossAnalysis2.ipynb

```diff
@@ -159,7 +159,7 @@
     "\n",
```

Comment:
Shouldn't "other_assets" now be ['TKN2']? I think as it stands this notebook is changing the price of TKN, not TKN2?

Reply:
You were right, fixed that.